coursework_JS_advanced/index.js

```diff
@@ -33,25 +11,35 @@ async function getData(url) {
         console.log(e.message)
     });
     const data  = await response.json();
-    if(url == urlKiev)
-        {
-            city[0].textContent = data.name;
-            cityTemp[0].innerHTML = 'TEMPERATURE: ' + Math.round(data.main.temp-273) + '&deg';
-            feelsLike[0].innerHTML = 'FEELS LIKE: ' + Math.round(data.main.feels_like-273) + '&deg'; 
-        }
-        else if (url == urlLondon)
-        {
-            city[1].textContent = data.name;
-            cityTemp[1].innerHTML = 'TEMPERATURE: ' + Math.round(data.main.temp-273) + '&deg';
-            feelsLike[1].innerHTML = 'FEELS LIKE: ' + Math.round(data.main.feels_like-273) + '&deg'; 
-        }
-        else if(url == urlNY)
-        {
-            city[2].textContent = data.name;
-            cityTemp[2].innerHTML = 'TEMPERATURE: ' + Math.round(data.main.temp-273) + '&deg';
-            feelsLike[2].innerHTML = 'FEELS LIKE: ' + Math.round(data.main.feels_like-273) + '&deg'; 
-        }       
+    console.log(data);
+    return data;  
 };
 
+class showInfo {
+    constructor(info, city, cityTemp, feelsLike) {
+        this.info = info;
+        this.city = city;
+        this.cityTemp = cityTemp;
+        this.feelsLike = feelsLike;
+    }
+    show() {

```

Comment:
Move this method out. The class should not be aware about html elements on a page